repo/meta/schema/package-schema.json

```diff
@@ -109,6 +109,11 @@
           "description": "Flag indicating if the package is selected in search results",
           "default": false
         },
+	"hasKnownIssues": {
```

Comment:
please fix formatting for this file and any others

Reply:
this might need to be deleted , because if we are getting new fields added to packaging version ie. v4 then would just need adding new field to package.json which is already done , ill clean it up , once last updated field PR gets merged

Reply:
oh yeah that's true; okay let's see after PR is merged